integrationTests/p2p/antiflood/blacklist/blacklist_test.go

```diff
@@ -2,12 +2,13 @@ package blacklist
 
 import (
 	"fmt"
-	"github.com/ElrondNetwork/elrond-go/integrationTests/p2p/antiflood"
 	"math"
 	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/ElrondNetwork/elrond-go/integrationTests/p2p/antiflood"
+
```

Comment:
remove empty line and run again goimports

Reply:
done